pkg/ui/console/progress.go

```diff
@@ -156,6 +156,15 @@ func (u *Progress) GetBar(name string) *Bar {
 	return bar
 }
 
+func (u *Progress) AbortAll() {
+	u.lock.RLock()
```

Comment:
What happens if (theoretically) another thread accesses this progress bar instance while we are aborting? The reason I ask is if we should use a general lock instead of a read lock?

Reply:
I think general lock would be better, i'll swap into that.